build/__pythonVersions.sh

```diff
@@ -9,4 +9,3 @@ PYTHON38_VERSION='3.8.12'
 PYTHON39_VERSION='3.9.7'
 PYTHON310_VERSION='3.10.4'
 PYTHON311_VERSION='3.11.0b1'
-
```

Comment:
Don't think we need these changes.

Reply:
after running the `generateconstants.sh` these changes are added. I will ignore this change.

Reply:
auto generated.